sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -168,5 +172,84 @@ internal static string GetOperationName(Activity activity, ref AzMonList MappedT
 
             return activity.DisplayName;
         }
+
+        private static void AddExceptionTelemetryFromActivityExceptionEvents(Activity activity, ref TagEnumerationState monitorTags, string roleName, string roleInstance, string instrumentationKey, List<TelemetryItem> telemetryItems)
+        {
+            // Export exceptions from server spans only
+            // TODO: how do we handler consumer/producer spans?
+            if (!activity.Events.Any() || activity.Kind != ActivityKind.Server)
```

Comment:
Exporter shouldn't ignore exceptions based on Kind.

Reply:
This is something I am planning to validate with Osvaldo. There is no client side exception thing in AI SDK. As an example if there is a network error while making Http call, AI SDK will not export that exception as a separate exception telemetry if it was handled by user using try catch. It will just export the dependency telemetry with resultCode as `Faulted`.

In case of exporter we do not have a way to know whether it was handled or not Because we simply look at the activity event and export it (HttpClient fires exception event and instrumentation library records it). 

This could be a behavior change between AI SDK and OTEL but it needs to be called out as we will export more data than we do currently with AISDK. That is the reason to limit the exception export to `Kind == Server` for now. If we agree with the behavior change, will remove the condition.

Reply:
> As an example if there is a network error while making Http call, AI SDK will not export that exception as a separate exception telemetry if it was handled by user using try catch. It will just export the dependency telemetry with resultCode as Faulted.

^ Same with Asp.net Core reporting RequestTelemetry (and not ExceptionTeleemtry)

Reply:
Removed the check as per offline discussion.